.gitignore

```diff
@@ -0,0 +1,24 @@
+config.json
+
+# Logs
+logs
+*.log
+npm-debug.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
```

Comment:
Are these comments in here for beginners or are they taken from  a template you have?